pkg/forwarder/internal/retry/on_disk_retry_queue.go

```diff
@@ -229,16 +228,21 @@ func (s *onDiskRetryQueue) reloadExistingRetryFiles() error {
 }
 
 func (s *onDiskRetryQueue) getExistingRetryFiles() ([]os.FileInfo, int64, error) {
-	entries, err := ioutil.ReadDir(s.storagePath)
+	entries, err := os.ReadDir(s.storagePath)
 	if err != nil {
 		return nil, 0, err
 	}
 	var files []os.FileInfo
 	currentSizeInBytes := int64(0)
 	for _, entry := range entries {
-		if entry.Mode().IsRegular() && filepath.Ext(entry.Name()) == retryTransactionsExtension {
-			currentSizeInBytes += entry.Size()
-			files = append(files, entry)
+		info, err := entry.Info()
+		if err != nil {
+			continue
+		}
```

Comment:
Let's not silent this one.

```suggestion
		if err != nil {
			log.Warn("Can't get file info", err)
			continue
		}
```